### ISSUE TITLE
systray.SetTitle("Awesome App") doesn't work

### ISSUE THREAD

**Issue by adeptoleg** [getlantern/systray#84]
Doesn't show text "Awesome App". OS WIN10 x64
```
GOROOT=E:\Go #gosetup
GOPATH=C:\Users\adept\go #gosetup
E:\Go\bin\go.exe build -i -ldflags -H=windowsgui -o E:\Projects\TestCompile\go_build_TestTimeGo_.exe . #gosetup
"E:\Program Files\JetBrains\GoLand 2018.3.5\bin\runnerw64.exe" E:\Projects\TestCompile\go_build_TestTimeGo_.exe -i -ldflags -H=windowsgui #gosetup
runnerw.exe: AttachConsole failed with error 6: The handle is invalid.
```

**Comment by joesis** [comment]
@adeptoleg from what you posted, it looks like a problem of the toolchain. Did you try build from the official Go Windows compiler?

**Comment by adeptoleg** [comment]
@joesis Yes. I'm trying a build from the official Go Windows compiler v1.12

**Comment by joesis** [comment]
@adeptoleg hmm we never encountered this. What's this `JetBrains\GoLand 2018.3.5` thing? Does your setup work well for other Go code, like a simple Hello world? If you saw the error from IDE, I'd suggest you to try building from command line first. IDE is not always your friend.

**Comment by adeptoleg** [comment]
@joesis I tried to build the same result from the command line. Other projects compile normally. Without text, it also works

**Comment by joesis** [comment]
@adeptoleg can you post more details you think may be useful, like the command history and the output? We may not be able to help you by any means without sufficient information.

**Comment by ergoz** [comment]
SetTitle not works on Windows. You can check source file systray_windows.go:L644 and find this:

```
// SetTitle sets the systray title, only available on Mac.
func SetTitle(title string) {
	// do nothing
}
```

**Comment by homaxweb** [comment]
Hi, is there a problem to implement this function?

**Comment by alourie** [comment]
I'm having this issue as well, running master branch from git. `SetTitle()` doesn't seem to have any effect.

I'm running Linux (arch) with i3 and polybar as a systemtray, also tested with trayer. Any pointers would be appreciated.

**Comment by henri9813** [comment]
No news about it ?
